exo-run/features/steps/steps-then.ls

```diff
@@ -2,32 +2,37 @@ require! {
   'async'
   'chai' : {expect}
   'child_process'
+  '../../../exosphere-shared' : {compile-service-routes}
+  'fs'
   'jsdiff-console'
   'nitroglycerin' : N
-  'prelude-ls' : {last}
+  'prelude-ls' : {any, last}
+  'os'
+  'path'
   'request'
+  'jsonic'
```

Comment:
We can get rid of this once this ticket has been resolved: https://github.com/Originate/exosphere-sdk/issues/279